src/pages/scheduler/components/Event.tsx

```diff
@@ -0,0 +1,22 @@
+import { Flex, Heading, HStack, VStack } from '@chakra-ui/react';
+import { EventProps } from 'react-big-calendar';
+
+export const CustomEvent = ({ title, event }: EventProps) => {
```

Comment:
the name `CustomEvent` seems a bit redundant and I'd say something like `CalendarEvent` is more clear